src/me/components/Resources.tsx

```diff
@@ -15,8 +15,9 @@ import {
   Heading,
   HeadingElement,
 } from '@influxdata/clockface'
-import AnnouncementCenter from './AnnouncementCenter'
```

Comment:
While you're in the file, would you mind changing the component name ("Resources") to something a little more descriptive re what it's doing? And that also clarifies it's a cloud component?

Since everything above it in the hierarchy is potentially an OSS component, you have to browse through the full history to figure out that this is actually a cloud-only component, but only because of the `CLOUD ?` check in HomePageContents.tsx and HomePageContentsTsm.tsx.

src/App.tsx --> SetOrg --> route for `HomePageContainer` --> HomepageContents / HomepageContentsTSM --> CLOUD check confirms this component isn't used in OSS.